src/components/SubHeader/subHeader.css

```diff
@@ -74,4 +81,11 @@
     line-height: 25px;
     letter-spacing: 0.2px;
     color: #FFFFFF;
-  }
\ No newline at end of file
+  }
+
+  .subHeader_screens .screen {
+    position: relative;
+    right: 25px;
+    margin-top: 30px;
```

Comment:
you can directly apply `width: a preceratage prefereably `   e.g  ` width: 80%`
this will shrink the image.

then on the div containing the image
```
display: flex;
justify-content: center;
padding: adjust to fit as necessary;
```
if you want, you can apply `margin: auto` to tidy it up even more